name:        "kayeton1"
cam_prod_id: 9760
width:       640
height:      480
fps:         90
